styles.css

```diff
@@ -98,7 +98,7 @@ body {
 
 .main-program {
   background-image: url(./images/main_bgrd.png);
-  background-color: #272a31;
+  background-color: #272a30;
```

Comment:
-  You have done great so far. :+1: :+1: Your Project looks like the provided design.  but there is an issue you need to fix before moving to the next milestone. :100: 
- [ ] Looking at the video below, you can see that when I open the mobile menu and click on an item, it takes me to that section but then the logo tends to appear and fill up the nav bar section. I'll suggest you fix this as it distorts the UI :+1: :smile: 

- - VIDEO FOR REFERNCE:

https://github.com/gdvl777/CapstoneSmash/assets/107632780/7c5bb621-de02-4a9f-a085-699a7c3baff4